design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,229 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer ([@rathpc](https://github.com/rathpc))
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
```

Comment:
can we list the proposed labels right here in this section? they are currently only listed inline in the examples and it would be nice to see them in a more prominent highlighted location, like here in this section